oci/pack.go

```diff
@@ -389,7 +387,7 @@ func (ocipack *ociPackage) Pull(ctx context.Context, opts ...pack.PullOption) er
 		return err
 	}
 
-	fullref := ocipack.Name()
+	fullref := fmt.Sprintf("%s:%s", ocipack.Name(), ocipack.Version())
```

Comment:
nit: I see that this format string is used in two places. We might as well create a very small function to encapsulate this format to ensure that all  functions use the same.

Reply:
I looked internally into [the `name` package](github.com/google/go-containerregistry/pkg/name) to see if we could re-use anything there but there was nothing from a first glance.  I've just done a fixup and will push the change now.